wpimath/src/main/native/cpp/geometry/struct/Ellipse2dStruct.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/geometry/struct/Ellipse2dStruct.h"
+
+namespace {
+constexpr size_t kCenterOff = 0;
+constexpr size_t kXSemiAxisOff = kCenterOff + wpi::GetStructSize<frc::Pose2d>();
+constexpr size_t kYSemiAxisOff = kXSemiAxisOff + sizeof(units::meter_t);
```

Comment:
We should be consistent about using numeric literals (like in `GetSize()` and other struct implementations) and sizeof.

Reply:
Does `GetSize()` actually work here?

Reply:
No, sorry for the confusion- What I meant was that the implementation of `GetSize()` uses a numeric literal (and the other struct implementations use numeric literals), unlike this code.